tests/README.md

```diff
@@ -0,0 +1,30 @@
+# Running tests against a pack
+
+This branch contains the UI tests for API Manager 3.2.0. See the following steps to run the tests against the required pack.
+
+1. Start the pack.
+2. Open the `tests/` directory.
```

Comment:
Better to say "Navigate to `tests/` directory.

Reply:
Added with https://github.com/wso2/apim-apps/pull/133/commits/a3b8c32d3eba06950156cb6bec7f52aea1a4b198